modules/rosa-sts-byo-oidc-options.adoc

```diff
@@ -1,11 +1,11 @@
 // Module included in the following assemblies:
 //
-// * rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc
-// * rosa_getting_started/quickstart.adoc
+//
```

Comment:
Tiny nit: I'd delete this line so there's only one blank comment line above the actual module names.